deployment/provision_vagrant_vm.sh

```diff
@@ -46,5 +45,6 @@ sudo -H -u vagrant yarn --no-bin-links
 # setup 1327
 cd /vagrant
 sudo -H -u vagrant python3 manage.py migrate --noinput
+sudo -H -u vagrant python3 manage.py shell -c "from django.contrib.auth import get_user_model; User = get_user_model(); User.objects.create_superuser('admin', email='admin@localhost', password='admin'); print('Created superuser')"
```

Comment:
you could make this more explicit as we know what our user model is (`UserProfile`).
also, please use a different email address with explicitly specified tld, so that the user form will accept changes to the user profile later on without form field errors.
it would also be great if you could add the admin user to a group and give this group the three add permissions for minutes, polls and information documents.